kyo-prelude/shared/src/main/scala/kyo2/kernel/Pending.scala

```diff
@@ -14,7 +14,28 @@ object `<`:
 
     implicit private[kyo2] inline def apply[A, S](p: Kyo[A, S]): A < S = new <(p)
 
-    implicit inline def lift[A, S](v: A)(using inline ng: NotGiven[A <:< (Any < Nothing)]): A < S = <(v)
+    implicit inline def lift[A, S](v: A)(using inline ng: NotGiven[A <:< (Any < Nothing)]): A < S =
+        <(v)
+
+    implicit inline def liftPureFunction1[A1, B](inline f: A1 => B)(
+        using inline ng: NotGiven[B <:< (Any < Nothing)]
```

Comment:
Does this need a name?

Reply:
yeah, the compiler requires it because of `inline`